Tests/FunctionalTests/Sources/FTRAnimationsTest.m

```diff
@@ -34,6 +35,18 @@ - (void)testUIViewAnimation {
       assertWithMatcher:grey_text(@"UIView animation finished")];
 }
 
+- (void)testAnimationSpeed {
```

Comment:
seems like this would be better as a unit test as it's doesn't require interaction with the UI.